packages/angular/src/generators/application/application.ts

```diff
@@ -46,6 +47,18 @@ export async function applicationGenerator(
     You can resolve this error by removing the "standalone" option or by migrating to Angular 14.1.0.`);
   }
 
+  if (
+    gte(installedAngularVersionInfo.version, '14.1.0') &&
+    schema.standalone === undefined &&
+    process.env.NX_INTERACTIVE === 'true'
+  ) {
+    schema.standalone = await prompt({
+      name: 'standalone-components',
+      message: 'Would you like to use Standalone Components?',
+      type: 'confirm',
+    });
```

Comment:
`prompt` returns an object with `name` as the key and the result as the value of that key. In this particular case it will return:

```js
{
  'standalone-components': true // or false
}
```

You need to get the value from that property.